Sources/Prometheus/MetricTypes/Counter.swift

```diff
@@ -67,17 +67,16 @@ public class Counter<NumType: Numeric, Labels: MetricLabels>: Metric, Prometheus
     ///     - amount: Amount to increment the counter with
     ///     - labels: Labels to attach to the value
     ///
-    public func inc(_ amount: NumType = 1, _ labels: Labels? = nil, _ done: @escaping (NumType) -> Void = { _ in }) {
+    public func inc(_ amount: NumType = 1, _ labels: Labels? = nil, _ done: @escaping () -> Void = { }) {
```

Comment:
We should keep the argument in the callback here, since it's a different value than the value passed in, and might be interesting to the user.